src/app/ingredients/ingredients.component.html

```diff
@@ -0,0 +1,7 @@
+<div class="ui-g ingredients-wrap">
+    <div class="ui-g-4 ui-md-3 ui-lg-2 ingredient-el" *ngFor="let ingredient of ingredients">
+      <img class="ingredient-img" [src]="'http://www.thecocktaildb.com/images/ingredients/' + ingredient.strIngredient1 +'-Medium.png'">
```

Comment:
maybe it would be better to move this concatenation as a getter inside the component?

Reply:
"http://" will not work on heroku. It will throw error: 'mixed content type'